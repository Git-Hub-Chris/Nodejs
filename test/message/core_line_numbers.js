'use strict';
require('../common');
const path = require('path');

// This test verifies that line numbers in core modules are reported correctly.
// The path module was chosen for testing because it changes infrequently.
// If this test begins failing, it is likely due to a path update, and the
// test's assertions simply need to be updated to reflect the changes. If a
// path update was not made, and this test begins failing, then line numbers
// are probably actually broken.
path.dirname(0);
