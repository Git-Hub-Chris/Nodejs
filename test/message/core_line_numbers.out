punycode.js:69
		throw new RangeError(I18N(errors[type]));
		^

RangeError: Invalid input
    at error (punycode.js:69:*)
    at Object.decode (punycode.js:*:*)
    at Object.<anonymous> (*test*message*core_line_numbers.js:*:*)
    at Module._compile (module.js:*:*)
    at Object.Module._extensions..js (module.js:*:*)
    at Module.load (module.js:*:*)
    at Function.Module._load (module.js:*:*)
    at Function.Module.runMain (module.js:*:*)
    at startup (node.js:*:*)
    at node.js:*:*
